kombu/transport/redis_cluster.py

```diff
@@ -299,7 +317,10 @@ def close(cls, connection):
             connection.close()
             del cls.refcounts[key]
             del cls.connection_to_key[connection]
-            del cls.connections[key]
+            if key in cls.producer_connections:
+                del cls.producer_connections[key]
+            else:
+                del cls.consumer_connections[key]
```

Comment:
minor comment. since key includes the key `consumer` or `producer`, can't we identify it without checking key's existence?